docs/contrib/Logger.rst

```diff
@@ -1,29 +1,94 @@
 Using the logging infrastructure
 ================================
 
-In order to ensure consistent messaging from the openroad application we
-have adopted spdlog as our logging infrastructure. We have a thin
-wrapper on top for extensibility. Whenever a message needs to be issued
-you will use one of the logging functions in the ‘ord’ namespace.
+OpenROAD provides spdlog as part of logging infrastructure in order to ensure a clear, consistent messaging and complete messaging interface. 
+A wrapper formats the prefix in the reccommended messaging style and limit. 
+A message format is as follows:
+<tool id> <message id> : <Message body>.
+
+For Example,
+DRT 0001 TritonRoute.cpp:333       Error reading param file: {}.
```

Comment:
This is how it appears in messages.txt but in the actual tool output you see something like

[INFO ODB-0127] Reading DEF file: ./results/asap7/aes/base/4_cts.def